.github/CODEOWNERS

```diff
@@ -0,0 +1,16 @@
+.github/            @dart-ecosystem-admins
+.gemini/            @dart-ecosystem-admins
+README.md           @dart-ecosystem-admins
+pkgs/args           @munificent
+pkgs/async          @dart-ecosystem-admins
+pkgs/characters     @lrhn
+pkgs/collection     @dart-ecosystem-admins
+pkgs/convert        @dart-ecosystem-admins
+pkgs/crypto         @dart-ecosystem-admins
+pkgs/fixnum         @dart-ecosystem-admins #?
+pkgs/lints          @dart-ecosystem-admins
+pkgs/logging        @dart-ecosystem-admins
+pkgs/os_detect      @lrhn
+pkgs/path           @munificent
```

Comment:
I'm one of the original authors, but I haven't maintained the path package in many years. I'd do `@dart-ecosystem-admins` for this one too.